packages/main/src/plugin/color-registry.ts

```
@@ -241,27 +241,44 @@ export class ColorRegistry {
+    if (themeId === 'hc-light' || themeId === 'hc-dark') {
+      return true;
+    } else if (themeId === 'light' || themeId === 'dark') {
+      return false;
+    } else {
+      const parent = this.#parentThemes.get(themeId);
+      if (parent) {
+        return this.isHighContrastTheme(parent);
+      } else {
+        return false;
+      }
+    }
```

Biome (2.5.3)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)